### PR TITLE
change: updated the admin api URI style.

### PR DESCRIPTION
**Pull request by membphis**
```
HTTP model management API:
    -- /apisix/admin/routes
    -- /apisix/admin/http/routes
    -- /apisix/admin/:workspace/http/routes
    -- /apisix/admin/routes/1
    -- /apisix/admin/http/routes/1
    -- /apisix/admin/:workspace/http/routes/1

STREAM model management API:
    -- /apisix/admin/stream/routes
    -- /apisix/admin/:workspace/stream/routes
    -- /apisix/admin/stream/routes/1
    -- /apisix/admin/:workspace/stream/routes/1

PLUGIN management API:
    -- /apisix/admin/plugins
    -- /apisix/admin/http/plugins
    -- /apisix/admin/plugins/udp-logger
    -- /apisix/admin/http/plugins/udp-logger
    -- /apisix/admin/stream/plugins
    -- /apisix/admin/stream/plugins/udp-logger

SCHEMA management API:
    -- /apisix/admin/schema/router
    -- /apisix/admin/schema/service
    -- /apisix/admin/http/schema/consumer
```

### CODE REVIEW

File: apisix/admin/init.lua [hunk 1]
```diff
@@ -256,23 +238,71 @@ local function reload_plugins(data, event, source, pid)
...
+        -- /apisix/admin/routes
+        uri_segs = {"", "apisix", "admin", "", "http", unpack_tab(uri_segs, 4)}
+
+    elseif valid_models[typ] then
+        -- /apisix/admin/http/routes
+        -- /apisix/admin/stream/routes/1
+        uri_segs = {"", "apisix", "admin", "", unpack_tab(uri_segs, 4)}
+
+    else
+        -- /apisix/admin/:workspace/http/routes
```
Comment (nic-chen) [inline apisix/admin/init.lua:276]: maybe we should add test case for this uri style ?

Comment (membphis) [reply apisix/admin/init.lua:276]: good catch, I will add test cases later

Comment (membphis) [reply apisix/admin/init.lua:276]: fixed already, please take a look when you have time

**Comment by membphis** [comment]
Ignore this PR, it seems that it is not necessary.

**Pull request closed without merging**